src/evaluators/utility/PHAL_SharedParameter.hpp

```diff
@@ -60,7 +71,7 @@ class SharedParameter : public PHX::EvaluatorWithBaseImpl<Traits>,
             this->registerSacadoParameter(param_name, paramLib);
             if (pvi.isParameter("Nominal Value")) {
               double nom_val = pvi.get<double>("Nominal Value");
-              value = nom_val;
+              accessor->getValue() = nom_val;
```

Comment:
If the problem created all the accessors, we would be able to remove this clunky initialization code from this evaluator, and stuff it in the problem class instead. I would add a method to AbstractProblem, like the following:

```
template<typename EvalT>
void addParameterAccessor (const std::string& name, const double default);
```
This method would do exactly what is done in SharedParameter (i.e., scan the "Paramater" section of the "Problem" list for a nominal value, and use the default if not found). Since the accessors are stored in the problem, it's conceptually more appropriate if the problem inits them.